python/tests/__init__.py

```diff
@@ -67,6 +68,7 @@ def skip_wrapper(*args, **kwargs):
         return test_item
     return decorator
 
+simulator_test = statoil_test
```

Comment:
This is just minor, but I'd get rid of `simulator_test`

`statoil_test` is well documented and communicates the requirements for running a test (ie, that statoil test data is available).
`simulator_test` I guess refers to what is being tested (the simulator). If someone does not know that it is just an alias for `statoil_test`, it does not provide much more information

Reply:
I have no strong feelings here; the point was that `@simulator_test` requires a simulator; and could in principle be run outside of Statoil as well. BUt of course - the actual data used in the test are still hardwired to Statoil / Equinor. I'll remove it.